### ISSUE TITLE
First start installs shell function even if confirmation question is answered with no

### ISSUE THREAD

**Issue by nkoester** [Canop/broot#81]
**Describe the bug**
Pretty much the title. Installed and started for the first time. Was unsure what would be installed so I chose 'no' and research what will be done. After inputting 'n' the installation simply went ahead:

``` bash
$ broot 

Broot should be launched using a shell function (see https://github.com/Canop/broot for explanations).
The function is either missing, old or badly installed.
Can I install it now? [Y n]
n
Writing br shell function in /home/nkoester/.local/share/broot/launcher/bash/1.
Creating link from /home/nkoester/.config/broot/launcher/bash/br to /home/nkoester/.local/share/broot/launcher/bash/1.
/home/nkoester/.bashrc successfully patched, you can make the function
immediately available with
source /home/nkoester/.bashrc
/home/nkoester/.zshrc successfully patched, you can make the function
immediately available with
source /home/nkoester/.zshrc
Writing br shell function in /home/nkoester/.local/share/broot/launcher/fish/1.fish.
Creating link from /home/nkoester/.config/fish/conf.d/br.fish to /home/nkoester/.local/share/broot/launcher/fish/1.fish.

The br function has been installed.
You may have to restart your shell or source your shell init files.
Afterwards, you should start broot with br in order to use its full power.
```

**To Reproduce**
1. install
2. make sure no prior config is in .config/broot
3. run broot command

**Expected behavior**
To not install anything if I chose no.

**Configuration (please complete the following information):**
 - OS: Linux 5.4.6-arch3-1 #1 SMP PREEMPT Tue, 24 Dec 2019 04:36:53 +0000 x86_64 GNU/Linux
 - Version broot 0.11.0 (installed via aur repository)

**Additional context**
Maybe give clear information *what* will be installed and added to various config files *before* giving the choice? I really dislike tools unexpectedly changing my configurations. Especially if I deny the request. The given URL links this repo and searching it for this information is cumbersome and his might discourage new users.

**Comment by Canop** [comment]
I pushed yesterday the fix for this but didn't make a version (I hadn't expected that broot would be featured on HN today...). I'll make the version now.

**Comment by Canop** [comment]
@nkoester  I just released the version 0.11.3 which should fix it
